### PR TITLE
Disable PoK popup if PoP is disabled

### PR DESCRIPTION
**Pull request by xackery**

### CODE REVIEW

**Review by joligario — APPROVED** [review]

File: global/global_player.lua [hunk 1]
```diff
@@ -336,7 +336,7 @@ function event_level_up(e)
     eq.popup("", "<c \"#F0F000\">Welcome to level 5.</c><br><br>You have just been granted a new ability called '<c \"#F0F000\">Origin</c>' which allows you to teleport back to your starting city.<br><br>Open the Alternate Advancement window by pressing the '<c \"#F0F000\">V</c>' key, look in the '<c \"#F0F000\">General' tab</c>, and find the '<c \"#F0F000\">Origin</c>' ability and select it.<br><br>Now press the '<c \"#F0F000\">Hotkey</c>' button to create a hotkey you can place on your hot bar.");
   end
 
-  if e.self:GetLevel() == 10 then
+  if e.self:GetLevel() == 10 and eq.is_the_planes_of_power_enabled() then
```
Comment (fryguy503) [inline global/global_player.lua:339]: PoK armor quests released with the updated Tutorial June 29th 2005 - http://www.tski.co.jp/baldio/patch/20050629.html

So this should be Dragons of Norrath flagged

Comment (xackery) [reply global/global_player.lua]: Changed. Let me know if you see anything else!

File: global/global_player.lua [hunk 2]
```diff
@@ -336,7 +336,7 @@ function event_level_up(e)
     eq.popup("", "<c \"#F0F000\">Welcome to level 5.</c><br><br>You have just been granted a new ability called '<c \"#F0F000\">Origin</c>' which allows you to teleport back to your starting city.<br><br>Open the Alternate Advancement window by pressing the '<c \"#F0F000\">V</c>' key, look in the '<c \"#F0F000\">General' tab</c>, and find the '<c \"#F0F000\">Origin</c>' ability and select it.<br><br>Now press the '<c \"#F0F000\">Hotkey</c>' button to create a hotkey you can place on your hot bar.");
   end
 
-  if e.self:GetLevel() == 10 then
+  if e.self:GetLevel() == 10 and eq.is_lost_dungeons_of_norrath_enabled() then
```
Comment (fryguy503) [inline global/global_player.lua:339]: eq.is_dragons_of_norrath_enabled()

Comment (xackery) [reply global/global_player.lua]: This correct?

**Pull request merged**